region/norfair/east/Green Bubbles Missile Room.json

```diff
@@ -369,6 +369,30 @@
       "collectsItems": [3],
       "flashSuitChecked": true
     },
+    {
+      "link": [2, 2],
+      "name": "G-Mode Setup - Get Hit By Geruta",
+      "requires": [
+        {"heatFrames": 550},
```

Comment:
https://videos.maprando.com/video/6823

The exact timing is pretty RNG.  Whether using your method or mine, the bat can get caught on the floor and ceiling.